service/controller/resource/ipam/resource.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	Persister          Persister
 }
 
+// Resource finds free IP ranges within a virtual network to create new subnets.
```

Comment:
> finds free IP ranges within a virtual network to create new subnets.

This is for node pools. For `AzureConfig` (clusters) it would be something like: "finds free IP ranges within an installation range to create new virtual network for the tenant cluster."

Reply:
Take a look now